backend/app/main.py

```diff
@@ -253,6 +258,53 @@ async def get_total_population(
             )
 
 
+@app.get("/api/document/{document_id}/{summary_stat}")
+async def get_summary_stat(
+    document_id: str, summary_stat: str, session: Session = Depends(get_session)
+):
+    try:
+        _summary_stat = SummaryStatisticType[summary_stat]
+    except ValueError:
+        raise HTTPException(
+            status_code=status.HTTP_400_BAD_REQUEST,
+            detail=f"Invalid summary_stat: {summary_stat}",
+        )
+
+    try:
+        stmt, SummaryStatsModel = {
+            "P1": (
+                text(
+                    "SELECT * from get_summary_stats_p1(:document_id) WHERE zone is not null"
+                ),
+                SummaryStatsP1,
+            ),
+            "P1TOTPOP": {
+                text("SELECT * from get_summary_stats_pop_totals(:document_id)"),
+                PopulationStatsP1,
+            },
```

Comment:
H: I don't think it makes sense to couple the document summaries and the pop summaries in an endpoint. Less code perhaps but I'd recommend we create a new endpoint for summaries which takes a gerrydb table name. This will have the added advantages of:
- having one less query in the UDF
- allowing us to share the "P1" id for both summary and document
- being able to get pop totals for the child layer if we ever need to do sum checks
- creating less coupling / being clearer what each endpoint does

Reply:
Just an example of where this coupling is confusing is `"results": [SummaryStatsModel.from_orm(row) for row in results],`; this doesn't really make sense – result totals shouldn't be iterable

Reply:
Ah, I see what you meant earlier now. Agreed and makes sense -- I can integrate this today